CMakeLists.txt

```diff
@@ -118,6 +118,13 @@ else()
 endif()
 
 if (CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64")
+  option(ENABLE_X86_HOST_DEBUG "Enables compiling on x86_64 host" FALSE)
+  if (NOT ENABLE_X86_HOST_DEBUG)
+    message(FATAL_ERROR
+    " FEX-Emu doesn't support compiling for x86-64 hosts!"
+    " This is /only/ a supported configuration for FEX CI and nothing else!"
+    " Pass -DENABLE_X86_HOST_DEBUG=True to bypass this message!")
```

Comment:
tbh drop this line, as long as CI knows the option better not tempt folk...

Reply:
Removed that line